cogs/weightcog.py

```diff
@@ -557,6 +560,45 @@ async def export(self, ctx: commands.Context, user: discord.Member = None) -> No
                 file=discord.File(csv_file, filename=f"{user.display_name}_weight_data.csv"),
             )
 
+    @commands.command()
+    async def del_all_log(self, ctx: commands.Context, user: discord.Member = None) -> None:
+        """
+        Deletes all log data (CSV file) associated with a specified Discord user or the command invoker.
+
+        Parameters:
+            ctx (commands.Context): The context of the command.
+            user (discord.Member, optional): The Discord user for whom to delete the log data. Defaults to the command invoker.
+
+        Returns:
+            None: This function does not return any value.
```

Comment:
Remove this. Doesn't add any value to tell it doesn't return anything.

Reply:
Removed.